backend/src/test/java/kr/touroot/travelogue/controller/TravelogueControllerTest.java

```diff
@@ -133,4 +134,47 @@ void findNotExistTravelogueThrowException() {
                 .statusCode(400).assertThat()
                 .body("message", is("존재하지 않는 여행기입니다."));
     }
+
+    @DisplayName("여행기를 삭제한다.")
+    @Test
+    void deleteTravelogue() {
+        Member member = testHelper.initMemberTestData();
+        testHelper.initTravelogueTestData(member);
+        String accessToken = jwtTokenProvider.createToken(member.getId());
+
+        RestAssured.given().log().all()
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + accessToken)
+                .when().delete("/api/v1/travelogues/1")
+                .then().log().all()
+                .statusCode(204);
+    }
```

Comment:
나중에 전체적으로 고쳐야 할 것 같기는 한데 testHelper가 initTravelogueTestData(member)를 할 때 생성된 id를 반환해줬으면 좋겠어요.

생성된 것을 지우려고 한다는 것이 명확하게 드러나도록요!

맡으신 업무 외적 작업이라 분담이 어려우시다면 공유 부탁드립니다!

```suggestion
    @DisplayName("여행기를 삭제한다.")
    @Test
    void deleteTravelogue() {
        Member member = testHelper.initMemberTestData();
        Travelogue saved = testHelper.initTravelogueTestData(member);
        String accessToken = jwtTokenProvider.createToken(member.getId());

        RestAssured.given().log().all()
                .header(HttpHeaders.AUTHORIZATION, "Bearer " + accessToken)
                .when().delete("/api/v1/travelogues/" + saved.getId())
                .then().log().all()
                .statusCode(204);
    }
```